model/project.go

```diff
@@ -437,11 +437,11 @@ type ContainerSystem struct {
 }
 
 type Module struct {
-	Name       string `yaml:"name,omitempty" bson:"name"`
-	Branch     string `yaml:"branch,omitempty" bson:"branch"`
-	Repo       string `yaml:"repo,omitempty" bson:"repo"`
-	Prefix     string `yaml:"prefix,omitempty" bson:"prefix"`
-	Ref        string `yaml:"ref,omitempty" bson:"ref"`
+	Name       string `yaml:"name,omitempty" bson:"name" plugin:"expand"`
```

Comment:
We need plugin here for expansions, right? Since this isn't something we typically expand, a comment would be helpful.